.rubocop.yml

```diff
@@ -132,3 +132,27 @@ Layout/SpaceInsideHashLiteralBraces:
 
 Style/RaiseArgs:
   Enabled: false
+
+Style/SlicingWithRange:
+  Enabled: true
+
+Lint/MixedRegexpCaptureTypes:
+  Enabled: true
+
+Lint/ShadowingOuterLocalVariable:
+  Enabled: true
+
+Layout/SpaceAroundEqualsInParameterDefault:
+  Enabled: true
+
+Naming/PredicateName:
+  Enabled: true
+
+Style/SymbolProc:
+  Enabled: true
+
+Style/RegexpLiteral:
+  Enabled: true
+
+Style/RedundantRegexpEscape:
+  Enabled: true
```

Comment:
Can you ensure there's a line ending? Most editors can be configured to do so. https://medium.com/@alexey.inkin/how-to-force-newline-at-end-of-files-and-why-you-should-do-it-fdf76d1d090e and https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline have different reasons why it's good.